prow/jobs/test-infra/buildpack.yaml

```diff
@@ -300,8 +300,8 @@ prow_clonerefs_job_template: &prow_clonerefs_job_template
           memory: 1.5Gi
           cpu: 0.8
 
-kyma_cleaners_job_template: &prow_tools_job_template
-  run_if_changed: "^development/tools/"
+prow_tools_job_template: &prow_tools_job_template
+  run_if_changed: "^development/tools/(pkg/|cmd/|Dockerfile)"
```

Comment:
I was actually going to suggest that we at least include `Gopkg.toml` with the regex, but it might be even better to 
- include go.mod as well, so we don't have to touch it again when we decide to upgrade tools to go.mod
- make it independent from folder structure

which led me to believe that this might be a better regex for this: `^development/tools/(.*\.go|.*\.toml|.*\.sh|go\.mod|Dockerfile|Makefile)$`

Reply:
https://regex101.com/r/GWwEkZ/1

Reply:
Yes, this definitely makes more sense. Will add that.